adal/src/main/java/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -1214,6 +1157,30 @@ public void startActivityForResult(Intent intent, int requestCode) {
         };
     }
 
+    private String mergeClaimsWithClientCapabilities(String claims) throws JSONException {
```

Comment:
can we change this function to package scope and write some tests to just test the input vs output in various scenarios. that way if we find a bug later we can just add to the unit test.